public/css/print.css

```diff
@@ -1,47 +1,49 @@
+@font-face {
+	font-family: 'Lato';
+	src: url('Lato/Lato-Regular.ttf') format('truetype');
+}
+
 body {
-	font-family: Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
-	font-size: 10px;
+	font-family: 'Lato', sans-serif;
+	font-size: 15px;
 }
 
 h1,
 h2,
 h3 {
 	margin: 0;
-	font-family: 'Roboto Slab', serif;
 }
 
 h1 {
 	width: 100%;
 	text-align: center;
-	margin-bottom: 1rem;
-	font-weight: 400;
-	font-size: 1.5rem;
+	margin: 1rem;
+	font-size: 25px;
```

Comment:
Totally me being OCD but it's maybe better to keep these alphabetical so in revisions it's easier to see what changes. And avoid accidentally doubling up on longer declarations.